SPECS/ostree/ostree.spec

```diff
@@ -1,25 +1,34 @@
 Summary:	Git for operating system binaries
 Name:		ostree
-Version:	2015.3
+Version:	2015.7
 Release:	1%{?dist}
-Source0:	http://ftp.gnome.org/pub/GNOME/sources/ostree/%{version}/%{name}-%{version}.tar.xz
+Source0:	http://ftp.gnome.org/pub/GNOME/sources/ostree/%{version}/%{name}-%{version}.tar.gz
 Source1:	91-ostree.preset
-Patch0:		ostree_syntax_error_fix.patch
+#Patch0:		ostree_syntax_error_fix.patch
 License:	LGPLv2+
 URL:		http://live.gnome.org/OSTree
 Vendor:		VMware, Inc.
 Distribution:	Photon
+BuildRequires:	git
 BuildRequires:	which
 BuildRequires:	libgsystem
 BuildRequires:	xz-devel
 BuildRequires:	gtk-doc
 BuildRequires:	e2fsprogs-devel
 Requires:	libgsystem
+Requires:	gpgme
+Requires:	libassuan
+Requires:	libgpg-error
+Requires:       systemd
 BuildRequires:	attr
 BuildRequires:	python2-libs
 BuildRequires:	python2
 BuildRequires:	gobject-introspection
+BuildRequires:	gobject-introspection-devel
 BuildRequires:	gobject-introspection-python
+BuildRequires:  gpgme-devel
+BuildRequires:  libcap
+BuildRequires:  systemd
```

Comment:
It does not seem to require that. There are only a few scripts and conf files getting added to the package here.